src/Features/SupportReleaseTokens/SupportReleaseTokens.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+namespace Livewire\Features\SupportReleaseTokens;
+
+use Livewire\ComponentHook;
+
+use function Livewire\on;
+
+class SupportReleaseTokens extends ComponentHook
+{
+    public static function provide()
+    {
+        // Use `snapshot-verified` to run the check before any component properties are hydrated
+        // but after the snapshot has been verified to ensure it hasn't been tampered with...
+        on('snapshot-verified', function ($snapshot) {
+            // The we've mounted a lazy component params container, don't
+            // check the release token as there won't be one...
+            if ($snapshot['memo']['name'] === '__mountParamsContainer') return;
```

Comment:
I wonder if there's a way to not need this here somehow - feels leaky

Reply:
Fixed - the issue was the faux mount params container wasn't receiving a release token, because it didn't ever dispatch a dehydrate method. I've fixed this and now we don't need this work around.